Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -135,11 +135,45 @@ public InputActionProperty rotationInput
             }
         }
 
+        [SerializeField]
+        InputActionProperty m_TrackingStateFlags;
+        public InputActionProperty trackingStateFlags
+        {
+            get => m_TrackingStateFlags;
+            set { m_TrackingStateFlags = value; }
+        }
+
+
+        /// <summary>
+        /// Bitflag enum which represents what data was set on an associated Pose struct
+        /// This matches UnityXRInputTrackingStateFlags 
+        /// </summary>
+
+        [Flags]
+        protected enum PoseDataFlags
```

Comment:
Should `UnityEngine.XR.InputTrackingState` from the XR Module be used instead since that's the actual tracking enum being used? That would require adding a dependency on the XR Module, but then users would have access to all of the information.